mkdocs/docs/contributing/overview.md

```diff
@@ -44,7 +44,7 @@ Absolutely everyone is welcome to come to any of our meetings. You never need an
 invite to join us. In fact, we want you to join us, even if you don’t have
 anything you feel like you want to contribute. Just being there is enough!
 
-You can find out more about our meetings [here](../../../getting-started/help). You don’t have to turn on your video. The first time you come, introducing yourself is more than enough.
+You can find out more about our meetings [here](../../getting-started/help). You don’t have to turn on your video. The first time you come, introducing yourself is more than enough.
```

Comment:
I think "You don’t have to turn on your video." is very strange to say. I also think "The first time you come, introducing yourself is more than enough." is also a bit strange.

I think it should just say something along the lines of "We welcome anyone and everyone to join!" or something like that.